universal/components/ContentList/component.js

```diff
@@ -47,35 +46,19 @@ function renderList ({
   )
 }
 
-const defaultList = [
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item',
-  'List Item'
-]
-
-function ContentList ({
-  hasTitle = true,
+const ContentList = ({
+  customClass,
+  hasTitle,
   title,
-  bulletList = true,
-  items = defaultList,
-  itemWidth = 'half'
-}: ContentListProps) {
+  bulletList,
+  items,
+  itemWidth
+}: ContentListProps) => {
   return (
-    <SectionFullWidth>
-      <SectionContainer>
-        {renderTitle({hasTitle, title})}
-        {renderList({bulletList, items, itemWidth})}
-      </SectionContainer>
-    </SectionFullWidth>
+    <div className={customClass}>
```

Comment:
Or just remove it. I'm trying to think of a use case where you'd need this, but 99% of the time it is going to be nested in another component, ie SectionContainer.

I'm also thinking about that markup. Not sure that the `<ul> or <ol>` needs to be nested in a `<div>` container.